driver/src/lib.rs

```diff
@@ -1,3 +1,11 @@
+// TODO(nlordell): The order in which these two crates get linked seems to
+//   matter (usure why). And when we remove `extern crate` statements and let
+//   cargo decide the order it leads to a linking error. So for now, until we
+//   figure out exactly why this is happening lets keep this these two `extern
+//   crate` statements so we successfully link.
+extern crate ethereum_tx_sign;
+extern crate web3;
+
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FMxqvW5HnJU3iE%2Fgiphy.gif&token=helping" data-gitmeme-token="helping"><img src="https://media2.giphy.com/media/MxqvW5HnJU3iE/giphy.gif" title="Created by gitme.me with /helping"/></a>

Thanks for the help with this!

@josojo this might have been related to the linking issue you sometimes saw when building the docker image.